tests/test_madewithlau.py

```diff
@@ -9,6 +11,13 @@ class TestMadeWithLauScraper(ScraperTest):
     def test_host(self):
         self.assertEqual("madewithlau.com", self.harvester_class.host())
 
+    @unittest.skip("canonical_url will not pass with testhtml (uses example.com)")
```

Comment:
My mistake for not suggesting this during #891 - I think we should adjust this message to something more like:

`canonical_url is not available from this webpage`

(that makes it clearer under what conditions it _should_ be possible to remove the `unittest.skip` decorator in future)

Reply:
I'll handle this separately after the rest is merged in, there are skip cases with this explanation in both batches of updates so i think it'll be better to separate into its own PR.